src/components/Profile/Profile.js

```diff
@@ -1,23 +1,43 @@
 import React, { useEffect, useState } from 'react';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { ExternalLink } from 'react-external-link';
+import { updateReservation } from '../../redux/Rockets/rocketSlice';
 import '../../styles/profile.css';
 
 const Profile = () => {
   const missions = useSelector((state) => state.mission.data);
+  const rockets = useSelector((state) => state.rockets.rocketsArray);
+  const [reservedRockets, setReservedRockets] = useState([]);
   const [mission, setMission] = useState([]);
   const [empty, setEmpty] = useState(false);
+
+  const dispatch = useDispatch();
+
   useEffect(() => {
     const joinedMisssion = missions.filter((item) => item.member === true);
     setMission(joinedMisssion);
     if (joinedMisssion.length === 0) {
       setEmpty(true);
     }
   }, []);
+
+  useEffect(() => {
+    setReservedRockets(rockets.filter((rocket) => rocket.reserved));
+  }, [rockets]);
+
```

Comment:
I think it is best practice to have just one useEffect per component, I would suggest that you kindly move your logic into the useEffect already declared.

Reply:
`useEffect()` is used to manage the states individually and it's normal to have more than one. Here's the [official documentation](https://reactjs.org/docs/hooks-effect.html) for more information.